components/organisation-unit-tree/src/organisation-unit-tree/use-root-org-data/patch-missing-display-name.js

```diff
@@ -0,0 +1,29 @@
+export const fromEntries = entries =>
+    entries.reduce(
+        (collection, [key, name]) => ({
+            ...collection,
+            [key]: name,
+        }),
+        {}
+    )
```

Comment:
`Object.fromEntries` [has good browser support](https://caniuse.com/?search=fromEntries) so we should be able to use that instead

Reply:
Afaik we still support IE11

Reply:
Not since November 2019 :stuck_out_tongue_winking_eye:  (https://github.com/dhis2/notes/pull/45/files & https://github.com/dhis2/app-platform/blob/master/cli/config/.browserlistrc.js#L5)

Reply:
Oh that's nice. Wondering about opera mini (I think @amcgee or @ismay mentioned that it's being used in Africa quite a bit). But I assume this app won't be used on mobile devices for a LONG time, so I'll use `Object.fromEntries`